packages/driver/cypress/e2e/cypress/location.cy.js

```diff
@@ -256,7 +306,7 @@ describe('src/cypress/location', () => {
   context('.create', () => {
     it('returns an object literal', () => {
       const obj = Location.create(urls.cypress, urls.signin)
-      const keys = ['auth', 'authObj', 'hash', 'href', 'host', 'hostname', 'origin', 'pathname', 'port', 'protocol', 'search', 'toString', 'originPolicy', 'superDomain']
+      const keys = ['auth', 'authObj', 'hash', 'href', 'host', 'hostname', 'origin', 'pathname', 'port', 'protocol', 'search', 'toString', 'origin', 'superDomainOrigin', 'superDomain']
```

Comment:
```suggestion
      const keys = ['auth', 'authObj', 'hash', 'href', 'host', 'hostname', 'origin', 'pathname', 'port', 'protocol', 'search', 'toString', 'superDomainOrigin', 'superDomain']
```

Reply:
should be fixed in [2448c3f](https://github.com/cypress-io/cypress/pull/23879/commits/2448c3fab0f7f72ff92f72e954dfba7062bff374)